Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM openshift3/nodejs-010-rhel7
+FROM registry.access.redhat.com/openshift3/nodejs-010-rhel7
 
 ADD . /opt/app-root/src/
 
 EXPOSE 8080
 
-CMD /bin/bash -c 'npm start'
\ No newline at end of file
+CMD sed -e 's/HYSTRIXDASHBOARDURLPORT/'"$HOSTIP":"$HYSTRIX_DASHBOARD_PORT"'/; s/ZIPKINQUERYURLPORT/'"$HOSTIP"':'"$ZIPKIN_QUERY_PORT"'/' index.html.unprocessed > index.html && /bin/bash -c 'npm start'
```

Comment:
Replaces the variables set in `index.html.processed` with their values from the environment variables, and saves the output to `index.html` which is then served to the client. I'm sure there are better ways to do this, comments?